apps/data-types.md

```diff
@@ -483,12 +482,13 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | - | - |  - |
 | productId | string | The unique identifier of the product. |
 | name | string | The name of the product. |
-| visible | boolean | Indicates if the product is displayed in the shop. |
+| visible | boolean | Indicates if the product is displayed in the shop.|
 | shortDescription | string | The short description of the product. |
 | deliveryPeriod | string | The average time of the product being delivered to the customer. By default, the delivery period is displayed in days, but this can be changed by the merchant. |
 | description | string | The description of the product. |
 | title | string | The page title of the product page. |
 | priceInfo | object of [priceInfo](page:apps-data-types#priceinfo) | Price information on the product. |
+| bulkPriceInfo | object of [bulkPriceInfo](page:apps-data-types#bulkpriceinfo) | Information about bulk prices on the product. |
```

Comment:
I'm not that much into the terminology of bulk prices... but isn't it a bulk price "of a product"?

Reply:
I rephrase it a bit.